ERROR 2025-07-04 03:15:31,885 exception_handler 4460 4916 <class 'rest_framework.exceptions.MethodNotAllowed'>: Method "PUT" not allowed.
ERROR 2025-07-04 14:59:48,594 exception_handler 17960 18128 <class 'rest_framework.exceptions.MethodNotAllowed'>: Method "PUT" not allowed.
ERROR 2025-07-04 15:09:02,364 exception_handler 10136 20344 <class 'rest_framework.exceptions.MethodNotAllowed'>: Method "PUT" not allowed.
ERROR 2025-07-04 15:09:22,884 exception_handler 2856 18700 <class 'rest_framework.exceptions.MethodNotAllowed'>: Method "PUT" not allowed.
ERROR 2025-07-04 15:12:45,116 exception_handler 15024 18312 <class 'rest_framework.exceptions.ValidationError'>: {'valid_till': [ErrorDetail(string='This field is required.', code='required')]}
ERROR 2025-07-04 15:12:45,116 exception_handler 15024 18312 Response data: {'valid_till': [ErrorDetail(string='This field is required.', code='required')]}
ERROR 2025-07-04 15:57:42,141 exception_handler 10156 20360 <class 'AttributeError'>: 'Settings' object has no attribute 'EMAIL_VERIFICATION_LENGTH'
ERROR 2025-07-04 15:57:42,141 exception_handler 10156 20360 Response data: {'type': "<class 'AttributeError'>", 'action': '<api.resources.v1.email_verification.EmailVerificationCodeView object at 0x0000013B9343ACF0>', 'messages': ["'Settings' object has no attribute 'EMAIL_VERIFICATION_LENGTH'"], 'status': 500, 'error': True}
ERROR 2025-07-04 16:10:20,290 exception_handler 19304 15320 <class 'django.db.utils.IntegrityError'>: null value in column "deleted_at" of relation "email_verification" violates not-null constraint
DETAIL:  Failing row contains (1, anonymous, anonymous, 2025-07-04 10:40:20.281274+00, 2025-07-04 10:40:20.281289+00, null, test001@test.com, UA7slHtcZPy4X8Tetf57DJPX6MEqN3cL, 2025-07-04 11:10:20.281036+00).

ERROR 2025-07-04 16:10:20,291 exception_handler 19304 15320 Response data: {'type': "<class 'django.db.utils.IntegrityError'>", 'action': '<api.resources.v1.email_verification.EmailVerificationCodeView object at 0x0000020A6E176CF0>', 'messages': ['null value in column "deleted_at" of relation "email_verification" violates not-null constraint\nDETAIL:  Failing row contains (1, anonymous, anonymous, 2025-07-04 10:40:20.281274+00, 2025-07-04 10:40:20.281289+00, null, test001@test.com, UA7slHtcZPy4X8Tetf57DJPX6MEqN3cL, 2025-07-04 11:10:20.281036+00).\n'], 'status': 500, 'error': True}
ERROR 2025-07-04 16:24:57,056 exception_handler 2808 18848 <class 'django.db.utils.IntegrityError'>: null value in column "deleted_at" of relation "email_verification" violates not-null constraint
DETAIL:  Failing row contains (1, anonymous, anonymous, 2025-07-04 10:54:57.049738+00, 2025-07-04 10:54:57.049746+00, null, test001@test.com, CpuTb8RQSRFKNBEUi0uzWbWpVKfZ9wkY, 2025-07-04 11:24:57.049557+00).

ERROR 2025-07-04 16:24:57,057 exception_handler 2808 18848 Response data: {'type': "<class 'django.db.utils.IntegrityError'>", 'action': '<api.resources.v1.email_verification.EmailVerificationCodeView object at 0x000002065D71ECF0>', 'messages': ['null value in column "deleted_at" of relation "email_verification" violates not-null constraint\nDETAIL:  Failing row contains (1, anonymous, anonymous, 2025-07-04 10:54:57.049738+00, 2025-07-04 10:54:57.049746+00, null, test001@test.com, CpuTb8RQSRFKNBEUi0uzWbWpVKfZ9wkY, 2025-07-04 11:24:57.049557+00).\n'], 'status': 500, 'error': True}
ERROR 2025-07-04 16:26:44,898 exception_handler 18600 7084 <class 'django.db.utils.IntegrityError'>: null value in column "deleted_at" of relation "email_verification" violates not-null constraint
DETAIL:  Failing row contains (1, anonymous, anonymous, 2025-07-04 10:56:44.896017+00, 2025-07-04 10:56:44.896028+00, null, test001@test.com, 2QVaJUWpuRVmjhF8Z2B7VUe1ObWhTfK2, 2025-07-04 11:26:44.895779+00).

ERROR 2025-07-04 16:26:44,898 exception_handler 18600 7084 Response data: {'type': "<class 'django.db.utils.IntegrityError'>", 'action': '<api.resources.v1.email_verification.EmailVerificationCodeView object at 0x000001E2D100ACF0>', 'messages': ['null value in column "deleted_at" of relation "email_verification" violates not-null constraint\nDETAIL:  Failing row contains (1, anonymous, anonymous, 2025-07-04 10:56:44.896017+00, 2025-07-04 10:56:44.896028+00, null, test001@test.com, 2QVaJUWpuRVmjhF8Z2B7VUe1ObWhTfK2, 2025-07-04 11:26:44.895779+00).\n'], 'status': 500, 'error': True}
ERROR 2025-07-04 16:28:33,946 exception_handler 20720 17656 <class 'rest_framework.exceptions.ValidationError'>: {'valid_till': [ErrorDetail(string='This field is required.', code='required')]}
ERROR 2025-07-04 16:28:33,946 exception_handler 20720 17656 Response data: {'valid_till': [ErrorDetail(string='This field is required.', code='required')]}
ERROR 2025-07-04 17:13:47,445 exception_handler 21244 18948 <class 'rest_framework.exceptions.ValidationError'>: {'valid_till': [ErrorDetail(string='This field is required.', code='required')]}
ERROR 2025-07-04 17:13:47,445 exception_handler 21244 18948 Response data: {'valid_till': [ErrorDetail(string='This field is required.', code='required')]}
ERROR 2025-07-04 17:56:44,330 exception_handler 18728 21540 <class 'rest_framework.exceptions.ValidationError'>: {'valid_till': [ErrorDetail(string='This field is required.', code='required')]}
ERROR 2025-07-04 17:56:44,331 exception_handler 18728 21540 Response data: {'valid_till': [ErrorDetail(string='This field is required.', code='required')]}
ERROR 2025-07-04 18:04:54,360 exception_handler 14008 22500 <class 'rest_framework.exceptions.ValidationError'>: [ErrorDetail(string='Invitation code or verification code is needed to create', code='invalid')]
ERROR 2025-07-04 19:00:51,686 exception_handler 22804 3112 <class 'rest_framework.exceptions.ValidationError'>: [ErrorDetail(string='Invitation code or verification code is needed to create', code='invalid')]
ERROR 2025-07-04 19:05:02,767 exception_handler 21584 16292 <class 'rest_framework.exceptions.ValidationError'>: [ErrorDetail(string='Invitation code or verification code is needed to create', code='invalid')]
ERROR 2025-07-04 19:07:35,867 exception_handler 11984 19568 <class 'rest_framework.exceptions.ValidationError'>: [ErrorDetail(string='Invitation code or verification code is needed to create', code='invalid')]
ERROR 2025-07-04 19:10:41,068 exception_handler 24720 24024 <class 'rest_framework.exceptions.ValidationError'>: [ErrorDetail(string='Invitation code or verification code is needed to create', code='invalid')]
